### PR TITLE
Various fixes and help with #5.

### PR DESCRIPTION
**Pull request by khatchad**
Please review and merge. You will need to upgrade to version 1.1.0 of https://github.com/khatchad/Common-Eclipse-Refactoring-Framework.

### CODE REVIEW

File: edu.cuny.citytech.foreachlooptolambda.ui/src/edu/cuny/citytech/foreachlooptolambda/ui/visitor/LambdaConversionVisitor.java [hunk 1]
```diff
@@ -3,160 +3,9 @@
...
- 	}
- 	public boolean visit(MethodRefParameter node) {
- 		return false; // comments
- 	}
- 	public boolean visit(MemberRef node) {
- 		return false; // comments
- 	}
- 	
+	public boolean visit(EnhancedForStatement node) {
+		return super.visit(node);
```
Comment (khatchad) [inline edu.cuny.citytech.foreachlooptolambda.ui/src/edu/cuny/citytech/foreachlooptolambda/ui/visitor/LambdaConversionVisitor.java:9]: @mdarefin You need to write code here.

Comment (mdarefin) [inline edu.cuny.citytech.foreachlooptolambda.ui/src/edu/cuny/citytech/foreachlooptolambda/ui/visitor/LambdaConversionVisitor.java:9]: for (int i =0; i <= node.getLength(); i++) {
            System.out.println(node);
        }
would it be print the node

Comment (khatchad) [inline edu.cuny.citytech.foreachlooptolambda.ui/src/edu/cuny/citytech/foreachlooptolambda/ui/visitor/LambdaConversionVisitor.java:9]: Why do you need a loop?

Comment (mdarefin) [inline edu.cuny.citytech.foreachlooptolambda.ui/src/edu/cuny/citytech/foreachlooptolambda/ui/visitor/LambdaConversionVisitor.java:9]: Under EnhancdForStatement: PARAMETER, EXPRESSION, BODY and those get divided into nodes, so if I just use a print statement, would it be print all?

Comment (khatchad) [inline edu.cuny.citytech.foreachlooptolambda.ui/src/edu/cuny/citytech/foreachlooptolambda/ui/visitor/LambdaConversionVisitor.java:9]: You tell me. What do you see in the output? Also, if it's a hint, the body of your loop is unchanged upon each iteration. Looks like you are doing the same thing over and over. Besides, I believe that `getLength()` returns the number of characters.

**Pull request merged**